mbs_results/config.json

```diff
@@ -129,6 +129,6 @@
 
     "census_extra_calibration_group": [5043, 5113, 5123, 5203, 5233,
      5403, 5643, 5763, 5783, 5903, 6073],
+     "filter_out_questions": [11, 12 ,146]
 
-    "calibration_group" : "calibration_group"
```

Comment:
This was highlighted from the hooks, not related to this task, `calibration_group` is defined twice in the config.json so removed the second to pass hooks